python/fapolicy_analyzer/ui/unapplied_changes_dialog.py

```diff
@@ -0,0 +1,22 @@
+import gi
+
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk, GLib
+from time import sleep
+from .ui_widget import UIWidget
+
+
+class UnappliedChangesDialog(UIWidget):
+    def __init__(self, parent=None, cancel_time=30):
```

Comment:
I'm guessing the `cancel_time` parameter here was a copy and paste leftover.

Reply:
You are correct. I missed that. I'll remove it.

Reply:
Pulled out that argument, as suggested.